src/robot/libraries/BuiltIn.py

```diff
@@ -613,9 +613,16 @@ def should_be_equal(self, first, second, msg=None, values=True):
         for example, string ``false`` or ``no values``. See `Boolean arguments`
         section for more details.
 
+        ``ignore_case`` is False by default.  It is a boolean value, and if True
+        indicates that 'first' and 'second' should be compared case-insensitively,
+        provided that 'first' and 'second' are string types.
+
         If both arguments are multiline strings, the comparison is done using
         `multiline string comparisons`.
         """
+        if ignore_case and is_string(first) and is_string(second):
```

Comment:
Should use `is_truthy(ignore_case)` similarly as other keywords accepting Boolean values. See also the previous comment about docs referring to `Boolean arguments` section.